server/src/main/java/org/activityinfo/ui/client/util/GwtUtil.java

```diff
@@ -135,4 +139,15 @@ public static Rectangle getBsContainerRectangle(com.google.gwt.user.client.Eleme
         }
         return null;
     }
+
+    public static void removeChildren(ComplexPanel panel) {
```

Comment:
Why is this needed rather than [ComplexPanel.clear()](http://www.gwtproject.org/javadoc/latest/com/google/gwt/user/client/ui/Panel.html#clear()) ?